.github/workflows/build-workflow.yml

```diff
@@ -58,6 +57,30 @@ jobs:
           name: debian-packages-${{ matrix.job.target }}
           path: target/${{ matrix.job.target }}/debian/*.deb
 
+  publish:
+    name: Publish ${{ matrix.job.target }}
+    runs-on: Ubuntu-20.04
```

Comment:
Just a note, maybe we can use the most latest version of Ubuntu? It could be the same even for other workflows as well.

Reply:
I would rather use something that does not change without us knowing about it (as this makes for a more predictable ci/cd process). If we use `ubuntu-latest` it might break unexpectedly when GitHub change what `latest` is actually resolved to.

Though any change to the `runs-on` field should be done in a separate PR...The majority of jobs use `ubuntu-20.04`, so I aligned with that pattern.